cwl_utils/cwl_v1_0_expression_refactor.py

```diff
@@ -216,7 +216,7 @@ def traverse(
     skip_command_line1: bool,
     skip_command_line2: bool,
 ) -> Tuple[Union[cwl.CommandLineTool, cwl.ExpressionTool, cwl.Workflow], bool]:
-    """Convert the given process and any subprocesess."""
```

Comment:
There were typos in the cwl files under the parser package, but there's a note at the top of those files saying they are auto-generated (i.e. better fix the typos at the source, if not already fixed). I noticed these other *expression_refactor files have similar typos, but not sure if these are also auto-generated? Happy to remove them from the PR if needed :+1:

Reply:
The expression refactor files are not autogenerated. So typo fixes are appreciated, thanks!